src/content/config.ts

```diff
@@ -45,6 +45,7 @@ const resourcesSchema = defineCollection({
 			'3rd-party',
 			'migration',
 		]),
+		tags: z.array(z.string()),
```

Comment:
Maybe we want to use a reference to a new `tags` collection to have proper display names?

Reply:
in a follow up pr